### ISSUE TITLE
Minimum revenue for selling

### ISSUE THREAD

**Issue by Fluorz** [TeamWertarbyte/crypto-trading-bot#48]
In this file : https://github.com/TeamWertarbyte/crypto-trading-bot/blob/development/src/modules/bot/index.ts

You are checking if revenue is more than 0 before selling. We can config a minimum revenue before selling.
Sometimes the bot is selling when he makes 10 cents, but with the bittrex commission we are losing money.

What do you think about minimum revenue before sell?

**Comment by Fluorz** [comment]
It will also be a good idea to set in then config the bittrex commission. Like this we are sure to sell when revenue - bittrex_commission > minimun_revenue

**Comment by saschb2b** [comment]
Good idea! Thanks for the suggestion. @Fluorz Will include that in the next release.
How should we interpret `minimun_revenue` ? As an absolute value like `5.50 -> 5$ and 50 cents` or percent like  `5.50 -> 5.5% of amountToInvest -> 5.5% from 50$ = 2.75$`

**Comment by Fluorz** [comment]
In percent I think : https://bittrexglobal.zendesk.com/hc/en-us/articles/360009625260-What-fees-does-Bittrex-Global-charge-

**Comment by saschb2b** [comment]
I meant the `minimun_revenue` parameter not `bittrex_commission`.
`bittrex_commission` seems to be just a percentage, yes.

**Comment by Fluorz** [comment]
`minimun_revenue`  in percent would be great 👍

**Comment by Fluorz** [comment]
A parameter to never reject and just hold would also be nice.

**Comment by saschb2b** [comment]
> A parameter to never reject and just hold would also be nice.
> 
> https://github.com/TeamWertarbyte/crypto-trading-bot/blob/development/src/modules/evaluation/EMAEvaluation.ts

Already implemented. See example for BTC and ETH here
https://github.com/TeamWertarbyte/crypto-trading-bot/blob/235120abbe806537c5d68a1d05107134e5d57801/config.json#L2

Just add your coins to hodl in that array

**Comment by Fluorz** [comment]
I meant globally

**Comment by saschb2b** [comment]
Would you mind creating a separate issue for that?

**Comment by Fluorz** [comment]
#49
